### DISCUSSION TITLE
Release date

### DISCUSSION THREAD

**Discussion by Wulfheart** [filamentphp/filament#28] [Help] [answered]
Hey,

if I remember correctly you wanted to release at March 1. Is this correct? 

![](https://media3.giphy.com/media/bIRFRUxcRG3Sw/giphy.gif)

**Comment by danharrin** [comment, marked as answer]
Yes! Tomorrow.

**Reply by Wulfheart** [reply]
![](https://media1.giphy.com/media/l3vR1AaADYcE7C8vu/giphy.gif)

**Comment by elmergustavo** [comment]
HI
